specification/cognitiveservices/Vision.ImageAnalysis/routes.tsp

```diff
@@ -30,10 +30,6 @@ op analyzeStream is Azure.Core.RpcOperation<
     @doc("A string indicating what visual feature types to return. Multiple values should be comma-separated. Valid visual feature types include: Tags, Caption, DenseCaptions, Objects, Read, SmartCrops, People. If VisualFeatures is not specified, then Categories, Tags, and Description are included in the response by default.")
     visualFeatures?: Array<visualFeatures>;
 
-    @doc("Impl Detail")
```

Comment:
Does the stream overload work without this? It's in the service swagger as being part of the route path...

https://github.com/Azure/azure-rest-api-specs/blob/1d7059db3c66d17d75e325d15c597ebbecfc60f4/specification/cognitiveservices/data-plane/UnifiedVision/preview/2023-04-01-preview/UnifiedVision.json#L3196C36-L3196C36

Reply:
Sorry I was sure this is something that was left over by mistake from your previous commit to clear up trying to overload the different operations. I will restore this line and find time later to look at the effect of this line in emitted code.